src/glib/mine/svm.h

```diff
@@ -224,17 +225,22 @@ inline TLinModel LibSvmSolveClassify(const TVec<TIntFltKdV>& VecV, const TFltV&
     // ignored by the learning when kernel_type!=polynomial
     svm_parameter.degree = 0;
     svm_parameter.C = Cost;
-    svm_parameter.nr_weight = 0;
-    svm_parameter.weight = NULL;
-    svm_parameter.weight_label = NULL;
+    svm_parameter.nr_weight = 2;
+    svm_parameter.weight_label = (int *)malloc(2 * sizeof(int));
```

Comment:
Where is this memory released?

Reply:
60 lines below:
svm_destroy_param(&svm_parameter);